examples/quick_start/ranking/README.md

```diff
@@ -0,0 +1,131 @@
+# Quick-start for ranking models with Merlin
```

Comment:
do you think this should go one level up? Like opening readme page right under [Merlin](https://github.com/NVIDIA-Merlin/Merlin/tree/quick_start_ranking)/[examples](https://github.com/NVIDIA-Merlin/Merlin/tree/quick_start_ranking/examples)/quick_start/ folder?

Reply:
It could be there, but maybe renamed to something like `ranking.md`?
The idea is that we gonna add other quick-start documents next, like Quick-start for session-based recommendation, for retrieval, for Two/Multiple Stages.
In that case, I think that should be a `README.md` that would work as an index for our quick-starts. What do you think?

Reply:
sounds good to me.

Reply:
Have created a [README.md](https://github.com/NVIDIA-Merlin/Merlin/tree/quick_start_ranking/examples/quick_start) with an intro to Quick-starts, and a link to the [ranking](https://github.com/NVIDIA-Merlin/Merlin/blob/quick_start_ranking/examples/quick_start/ranking.md) one